src/test/java/org/cactoos/http/HtResponseTest.java

```diff
@@ -53,7 +55,11 @@ public void worksFine() throws IOException {
                         new JoinedText(
                             "\r\n",
                             "GET / HTTP/1.1",
-                            String.format("Host:%s", home.getHost())
+                                new UncheckedText(
```

Comment:
@alex-semenyuk Why `UncheckedText` here? this test already declares it throws an `IOException`

Reply:
Fixed